jin/week1.js

```diff
@@ -0,0 +1,16 @@
+var numUniqueEmails = function (emails) {
+    let set = new Set();
+    for (let s of emails) {
+        let domain = '@' + s.split("@")[1];
+        let plusOmitted = s.split("+")[0];
```

Comment:
what if the email address doesn't have plus?
example@google.com
plusOmitted will be the whole string 'example@google.com', and this will be later delete dot to be 'example@googlecom'and  concatenated with domain, and result in 'example@googlecom@google.com'